snpm_cp.m

```diff
@@ -782,6 +798,45 @@ function snpm_cp(CWD)
 %-Loop over planes (breaks out after first loop if bVolm)
 %-----------------------------------------------------------------------
 nP = [];
+
+if(UseRapidPT >= 1)
```

Comment:
Added MinT computation to code. Second col of MaxT is -MinT.
See: https://github.com/nicholst/SnPM-devel/pull/28/commits/2f3c7f99f6ca9365adaba736cde007a7c9d6acd2